pkg/commands/generate/get_command.go

```diff
@@ -18,25 +18,24 @@ package generate
 import (
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/generate"
 	"github.com/spf13/cobra"
-	log "k8s.io/klog"
 )
 
 // GetCommand generate placeholder components
 func GetCommand() *cobra.Command {
 	var filepath string
 	var componentName string
+
 	cmd := &cobra.Command{
 		Use:   "generate",
 		Short: "Generate placeholder components",
 		Long:  "Generate placeholder components",
 		Run: func(cmd *cobra.Command, _ []string) {
-			if filepath == "" {
```

Comment:
Shouldn't we check this?

Reply:
I changed it to rely on the CWD by default (by essentially allowing an empty filepath).